js/src/components/paid-ads/validateCampaign.js

```diff
@@ -23,6 +23,26 @@ const validateCampaign = ( values ) => {
 		);
 	}
 
+	if (
```

Comment:
@dsawardekar I understand how you want to pass the budget and budgetMin values to `initialValues` to the `Form` component in `js/src/setup-mc/setup-stepper/setup-paid-ads/paid-ads-setup-sections.js` so that you can access them here. See my comment later below about that.
How about we don't pass `budget` and `budgetMin` to `initialValues` but as an object as second parameter to the `validateCampaign` function?